CONTRIBUTING.md

```diff
@@ -114,6 +115,9 @@ Consequently, is is quite normal to see warnings such as `WARNING: Could not
 compile cuda extensions` when building from source if you do not have CUDA
 available.
 
+Install the `docs` extras as well, if you are modifying the documentation / example
+notebooks.
```

Comment:
I think this comment would be better placed around line 108, describing the `docs` extra. It is already included as an extra to be installed on line 99.

Reply:
Yea good call, this was added before the semi-recent changes that you made. I'll remove these line then